test/micro/org/openjdk/bench/java/text/DefFormatterBench.java

```diff
@@ -43,32 +43,40 @@
 
 @BenchmarkMode(Mode.Throughput)
 @OutputTimeUnit(TimeUnit.MILLISECONDS)
-@Warmup(iterations = 5, time = 5, timeUnit = TimeUnit.SECONDS)
-@Measurement(iterations = 5, time = 10, timeUnit = TimeUnit.SECONDS)
-@Fork(1)
+@Warmup(iterations = 5, time = 1)
+@Measurement(iterations = 5, time = 1)
+@Fork(3)
 @State(Scope.Benchmark)
 public class DefFormatterBench {
 
-    @Param({"1.23", "1.49", "1.80", "1.7", "0.0", "-1.49", "-1.50", "9999.9123", "1.494", "1.495", "1.03", "25.996", "-25.996"})
-    public double value;
+    public double[] values;
 
-    private DefNumerFormat dnf = new DefNumerFormat();
+    @Setup
+    public void setup() {
+        values = new double[] {
+            1.23, 1.49, 1.80, 1.7, 0.0, -1.49, -1.50, 9999.9123, 1.494, 1.495, 1.03, 25.996, -25.996
+        };
+    }
+
+    private DefNumberFormat dnf = new DefNumberFormat();
 
     @Benchmark
     public void testDefNumberFormatter(final Blackhole blackhole) {
-        blackhole.consume(this.dnf.format(this.value));
+        for (double value : values) {
```

Comment:
Should this now include `@OperationsPerInvocation(13)` since the loop is performing multiple operations per invocation?